hack/csi_client/pkg/csi/client.go

```diff
@@ -24,9 +24,11 @@ const (
 
 var (
 	goOs          = runtime.GOOS
+	errVolumeAttr = fmt.Errorf("volume attributes are required")
```

Comment:
If we want this to be generic, then I think volumeattr and secrets should be optional

Reply:
Then should I remove `validateRequest`? That won't be doing much.

Reply:
Looks like some of these vars are now unused - delete them?